src/util/is-connected.ts

```diff
@@ -0,0 +1,17 @@
+import { Subscription } from "models/subscription";
+import { GitHubServerApp } from "models/github-server-app";
+import { Request } from "express";
+import { getConnectionsAndInstallations } from "routes/jira/jira-get";
+
+
+export const isConnected = async (req: Request, jiraHost: string, installationId: number): Promise<boolean> => {
+	const subscriptions = await Subscription.getAllForHost(jiraHost) || [];
+	const gheServers: GitHubServerApp[] = await GitHubServerApp.findForInstallationId(installationId) || [];
```

Comment:
Tbh, I don't think `GitHubServerApp.findForInstallationId(installationId)` is safe, as the `installationId` can be overlap between different customers